src/library/scala/collection/IterableOnce.scala

```diff
@@ -1198,7 +1201,11 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
 
   /** Convert collection to array. */
   def toArray[B >: A: ClassTag]: Array[B] =
-    if (knownSize >= 0) copyToArray(new Array[B](knownSize), 0)
+    if (knownSize >= 0) {
+      val a = new Array[B](knownSize)
```

Comment:
Do you mind giving `a` a more meaningful name? e.g. `copied` or `written`.

Reply:
👍